loaders/course_content_loader.py

```diff
@@ -53,19 +53,21 @@ def main():
 
     with session_factory.begin() as session:
         for eachRecord in records:
-            visible = True if eachRecord['visible'] == '1' else False
-            eachRecord['visible'] = visible
-            eachRecord['term'] = term
-            insert_stmt = insert(CourseContent).values(**eachRecord)
+
+            record_data = {
+                'content_id': eachRecord['content_id'],
+                'section': eachRecord['section'],
+                'activity_name': eachRecord['activity_name'],
+                'lesson_page': eachRecord['lesson_page'],
+                'visible': eachRecord['visible'] == '1',
+                'term': term,
+                'updated_at': datetime.utcnow(),
+            }
+
+            insert_stmt = insert(CourseContent).values(**record_data)
             do_update_stmt = insert_stmt.on_conflict_do_update(
                 index_elements=['term', 'content_id'],
-                set_=dict(
-                    section=eachRecord['section'],
-                    activity_name=eachRecord['activity_name'],
-                    lesson_page=eachRecord['lesson_page'],
-                    visible=visible,
-                    updated_at=datetime.utcnow()
-                )
+                set_=record_data
```

Comment:
Just to be clear + safe, let's keep the original pattern of only specifying fields we want to update. You can probably just modify the original version to refer to your `record_data`:

```python
            do_update_stmt = insert_stmt.on_conflict_do_update(
                index_elements=['term', 'content_id'],
                set_=dict(
                    section=record_data['section'],
                    activity_name=record_data['activity_name'],
                    lesson_page=record_data['lesson_page'],
                    visible=record_data['visible'],
                    updated_at=datetime.utcnow()
                )
            )
```